test/debugger.sh

```diff
@@ -5,25 +5,19 @@
 set -e
 
 project_root=`dirname $0`/..
-fixtures_dir=$project_root/fixtures
-declare -a test_apps=($fixtures_dir/jar-app $fixtures_dir/servlet-app)
+sample_app="$project_root/fixtures/sample-app"
+declare -a runtimes=("openjdk" "jetty")
 
 source $project_root/helpers.sh
 
+# TODO grep this from deploy output
 project=`get_active_project`
 project_url_suffix="$(echo $project | tr _ -).appspot.com"
 project_version="debugger-test"
```

Comment:
This may be more appropriate for a separate PR, but maybe we should use $runtime as the project_version, and also we should delete the version after the test is done - gcloud versions delete.